packages/components/src/LoadingOverlay.tsx

```diff
@@ -52,14 +50,13 @@ function LoadingOverlay({
                   data-testid={spinnerTestId}
                 />
               )}
-              {!isLoading && errorMessage != null && (
-                <FontAwesomeIcon icon={vsWarning} />
-              )}
             </div>
             {errorMessage != null && (
-              <div className="message-text" data-testid={messageTestId}>
-                {errorMessage}
-              </div>
+              <ChartErrorOverlay
+                errorMessage={errorMessage}
+                clearError={clearError}
+                data-testid={dataTestId}
+              />
```

Comment:
`LoadingOverlay` is built as a generic component; `ChartErrorOverlay` is specific to charting. `LoadingOverlay` should not depend on `ChartErrorOverlay`.